src/components/ErrorCard.tsx

```diff
@@ -0,0 +1,26 @@
+import { Dispatch, SetStateAction } from "react";
+
+type ErrorCard = {
+  error: Error | null;
```

Comment:
Im not sure if this is actually right, if theres no `error` instance, whats the purpose of this component then?

Perhaps is better to render at all from parent side, otherwise we will have a rendered block without error reference no?

Reply:
I was thinking of using this component for errors like on the server side, like misplacing the email and other things